src/components/molecules/widgets/RainAccidentsTableWidget.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import TableView from '../Table/TableView';
+import { IWidgetRainAccidentsBySeverityTableData } from '../../../models/WidgetData';
+import { createTableData } from '../Table/formatTableData.service';
+
+interface IProps {
+  data: IWidgetRainAccidentsBySeverityTableData;
+}
+
+const RainAccidentsTableWidget: FC<IProps> = ({ data }) => {
+  const { t } = useTranslation();
+  const { items, text } = data;
+
+  const labels: Array<string> = t('table.rainAccidentsLabels', { returnObjects: true });
+  const dataKeys = ['severity_hebrew', 'amount_of_accidents_caused_by_rain', 'accidents_caused_by_rain_percentage'];
```

Comment:
Using values like `severity_hebrew` makes the component very specific. we'll need to support multiple languages in the future.

Reply:
Now realized it is what the API returns.. I'll talk to @atalyaalon